batavia/modules/time.js

```diff
@@ -239,21 +248,28 @@ time.localtime = function(seconds) {
         throw new exceptions.TypeError.$pyclass('localtime() takes at most 1 argument (' + arguments.length + ' given)')
     }
     var date
-    if (arguments.length === 1) {
+    if (arguments.length === 1)
+    {
         // catching bad types
-        if (types.isinstance(seconds, [types.Complex])) {
+        if (types.isinstance(seconds, [types.Complex])) 
+        {
             throw new exceptions.TypeError.$pyclass("can't convert " + type_name(seconds) + ' to int')
-        } else if (!(types.isinstance(seconds, [types.Int, types.Float, types.Bool]))) {
+        }
+        else if (!(types.isinstance(seconds, [types.Int, types.Float, types.Bool]))) 
+        {
             throw new exceptions.TypeError.$pyclass('an integer is required (got type ' + type_name(seconds) + ')')
         }
 
         date = new Date(seconds * 1000)
-        if (isNaN(date)) {
+        if (isNaN(date))
+        {
             // date is too large per ECMA specs
             // source: http://ecma-international.org/ecma-262/5.1/#sec-15.9.1.1
             throw new exceptions.OSError.$pyclass('Value too large to be stored in data type')
         }
-    } else if (seconds === undefined) {
+    } 
+    else if (seconds === undefined) 
+    {
         date = new Date()
     }
 
```

Comment:
At column 6: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)